egs2/magicdata/asr1/README.md

```diff
@@ -0,0 +1,18 @@
+# RESULTS
+
+## Environments
+
+- date: `Mon Sep 19 20:16:36 EDT 2022`
+- python version: `3.9.12 (main, Apr  5 2022, 06:56:58)  [GCC 7.5.0]`
+- espnet version: `espnet 202207`
+- pytorch version: `pytorch 1.8.1+cu102`
+- Git hash: `30d08e87dfdfdc35ad6f438f01795c3efa531acf`
+    - Commit date: `Mon Sep 19 15:31:07 2022 -0400`
+
+## [Conformer](conf/tuning/train_asr_conformer.yaml) with [Transformer-LM](conf/tuning/train_lm_transformer.yaml)
```

Comment:
It would be great if you can also upload the model into Huggingface.

Reply:
You can finish that in another PR

Reply:
Got it. I'll train a couple of more epochs just to be safe, upload the model to HF, and update README.md in another PR.